utils/system.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import re
+
+# supposed to work on windows
+def parse_params():
```

Comment:
This is a non-standard way of handling cmd line args. Mind using argparse? 
https://docs.python.org/2/library/argparse.html